recsyslearn/dataset/segmentations.py

```diff
@@ -69,7 +69,7 @@ def segment(cls, dataset: pd.DataFrame, proportions=None, min_interaction: int =
             raise SegmentationNotSupportedException(
                 "Number of supported group is between 1 and 3.")
 
-        if np.sum(proportions * 10) / 10 != 1:
```

Comment:
Maybe we should change this everywhere? For instance in [ActivitySegmentation](https://github.com/giuliowaitforitdavide/recsyslearn/blob/a252afa822eb5ce30037ca29e61f5dd37ddba6ca/recsyslearn/dataset/segmentations.py#LL185C1-L186C46) it is still 
```
        if np.sum(proportions * 10) / 10 != 1:
            raise WrongProportionsException()
```